tests/components/geo_json_events/__init__.py

```diff
@@ -5,15 +5,19 @@
 
 def _generate_mock_feed_entry(
     external_id: str,
-    title: str,
+    title: str | int,
     distance_to_home: float,
     coordinates: tuple[float, float],
     properties: dict[str, Any] | None = None,
 ) -> MagicMock:
     """Construct a mock feed entry for testing purposes."""
     feed_entry = MagicMock()
     feed_entry.external_id = external_id
-    feed_entry.title = title
+    if isinstance(title, int):
+        # Convert integer to string to avoid slugify issues with the entity name
+        feed_entry.title = str(title)
+    else:
+        feed_entry.title = title
```

Comment:
I tried out some of my own testing and think this can be reverted if you apply my other suggestion.

Reply:
I will do that.